pkg/manager/member/pump_member_manager.go

```diff
@@ -470,6 +470,7 @@ func getNewPumpStatefulSet(tc *v1alpha1.TidbCluster, cm *corev1.ConfigMap) (*app
 
 			Template:             podTemplate,
 			VolumeClaimTemplates: volumeClaims,
+			PodManagementPolicy:  spec.PodManagementPolicy(),
```

Comment:
The `podManagementPolicy` is not set before as @july2993 has said in another comment, so the default value is `OrderedReady` for pump, and this field is not allowed to update after the sts is created, so we have to set to `OrderedReady` for pump if it's not set explicitly.